examples/CustomEntitlementComputationSample/gradle/libs.versions.toml

```diff
@@ -33,5 +33,4 @@ navigation-compose = "androidx.navigation:navigation-compose:2.5.3"
 compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling" }
 compose-ui-test-manifest = { module = "androidx.compose.ui:ui-test-manifest" }
 
-revenuecat = { module = "com.revenuecat.purchases:purchases", version.ref = "purchases" }
-revenuecat-amazon = { module = "com.revenuecat.purchases:purchases-store-amazon", version.ref = "purchases" }
```

Comment:
why did we have amazon here? 
also, don't we have to specify the flavor?

Reply:
Amazon now only has the `defaults` flavor, not the `customEntitlementComputation` flavor. But yeah, I don't think it's needed in this sample app.